packages/modules/good_we/device.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+from typing import Dict, List, Union, Optional
+
+from helpermodules.log import MainLogger
+from helpermodules.cli import run_using_positional_cli_args
+from modules.common import modbus
+from modules.common.abstract_device import AbstractDevice
+from modules.common.component_context import SingleComponentUpdateContext
+from modules.good_we import bat
+from modules.good_we import counter
+from modules.good_we import inverter
+
+
+def get_default_config() -> dict:
+    return {
+        "name": "GoodWe",
+        "type": "good_we",
+        "id": 0,
+        "configuration":
+        {
+            "ip_address": "192.168.193.15",
```

Comment:
Bitte keine Standard-IP vorgeben. Schon gar nicht eine, die potentiell vom EVU-Kit belegt ist.
Besser "None" nutzen, wie es auch bei anderen Modulen in 2.0 gemacht wird. Der zwingend erforderliche Standardwert für 1.9 kommt durch die updateConfig.sh ins System.